automation/taskcluster/androidTest/flank-x86.yml

```diff
@@ -44,7 +44,7 @@ flank:
   max-test-shards: -1 
   # repeat tests - the amount of times to run the tests.
   # 1 runs the tests once. 10 runs all the tests 10x
-  repeat-tests: 1
+  num-test-runs: 1
```

Comment:
Let's adjust the comment above or remove it.

Reply:
Changed it as in Fenix